src/routes/StackView/StackView.scss

```diff
@@ -9,9 +9,16 @@ $stack-view__parent-note-height: 22%;
   box-shadow: inset 0 0 0 $column__border-width var(--accent-color);
   background: rgba(var(--accent-color-rgb), 0.42);
 }
+[theme="dark"] {
+  .stack-view__portal {
+    backdrop-filter: blur(10px) brightness(0.24) saturate(0);
+    box-shadow: inset 0 0 0 $column__border-width var(--accent-color--dark);
+    background: rgba(var(--accent-color-rgb), 0.1);
+  }
+}
 
 .stack-view__portal-moderation-visible {
-  box-shadow: inset 0 0 0 $column__border-width $color-goal-green;
+  box-shadow: inset 0 0 0 $column__border-width $color-goal-green !important;
```

Comment:
Using `!important` is a bad practice and generally discouraged. Increase the specificity by using another class.
`.stack-view__portal.stack-view__portal-moderation-visible {` instead of `.stack-view__portal-moderation-visible {` works perfectly fine.